src/app/js/admin/enrichment/EnrichmentLogsDialog.js

```diff
@@ -87,6 +87,15 @@ export const EnrichmentLogsDialog = ({
 }) => {
     const [logsContainerRef, { width }] = useMeasure();
 
+    const handleDownloadLogs = () => {
+        const file = new Blob(logs, { type: 'text/plain' });
+        const element = document.createElement('a');
+        element.href = URL.createObjectURL(file);
+        element.download = 'enrichment-logs-' + Date.now() + '.log';
+        document.body.appendChild(element);
+        element.click();
```

Comment:
I'm not sure that this solution work on chromium, could you try it ?

Reply:
Successfully tested on Chrome 👍

Reply:
You probably want to make it invisible, or to remove it from the DOM once clicked

Reply:
I think it is better to create the file on click because you can open the log dialog at the beginning of the enrichment process. If you create the file at the render, the logs could be not complete.

Reply:
Can we still remove `element` from `body` once we're done though?